cloudformation/mongo-opsmanager-backup.template

```diff
@@ -357,7 +438,10 @@
               [
                 "#!/bin/bash -ev",
                 { "Fn::Join": [ "", ["/opt/features/ebs/add-encrypted.sh -s 100 -d f -m /backup -o 'defaults,noatime' -x  -t io1 -i ", {"Ref":"IOPS"}, " -k ", { "Ref": "CustomerMasterKey" }] ] },
-                "/opt/features/mongo-opsmanager/backup-agent-configure.sh"
+                "/opt/features/mongo-opsmanager/backup-agent-configure.sh",
+                "chown mongo-backup /opt/features/mongo-opsmanager/scripts/snapshot_backup.rb",
+                { "Fn::Join": [ "", ["echo '*/30 * * * * /opt/features/mongo-opsmanager/scripts/snapshot_backup.rb -b ", {"Ref":"SnapshotBackupBucketName"}, " -k ", { "Ref": "SnapshotBackupPublicKeysBucketName" }, "' > /opt/features/mongo-opsmanager/backup-agent-cron-job.txt"] ] },
+                "crontab -u mongo-backup /opt/features/mongo-opsmanager/backup-agent-cron-job.txt"
```

Comment:
It would be better if these lines were rolled into the backup-agent-configure script. General rule of thumb should be to make the CFN as abstract as possible and move the complexity into the scripts in the image.